assets/sample_docs/syntax.json

```diff
@@ -0,0 +1,9 @@
+{
+  // some comment
+  "internal_command": "#ff0000|#ffffff|b",
+  // some other comment
+  "external_command": "#ff0000|#ffffff|n",
+  "sub_command": "#ff0000|#ffffff|bli",
+  "keyword": "|#ffffff|u",
+  "punctuation": "#ff0000||s"
```

Comment:
Thinking through this a bit, now that I see it, would it make sense to split these strings into their parts?

```
"sub_command": {
  "bg": "#ff0000",
  "fg": "#ffffff",
  "attr": "n"
}
```

I think mostly people would copy paste to add new ones

Reply:
i was trying to keep it on one line per key and not have a sprawling json file. i tried that before and it was just a mess and hard to keep up with. if you think about most configs it key value but our will offer much more with fg anf bg and over a half dozen attributes. let's try to keep it small.

Reply:
i updated all this mess just because I like you. :)